### PR TITLE
Fix sort testcase input sorted unintentionally

### PR DESCRIPTION
**Pull request by vzvu3k6k**
As `getSortedVersion` and other sorting functions modify a given slice, testcase inputs are unintentionally sorted beforehand.

To avoid that, passes a cloned slices to these functions.

### CODE REVIEW

**Review by tjgurwara99 — APPROVED** [review]
All seems good to me. Maybe add a comment on top of `getSortedVersion` and `cloneIntSlice` to make it easier to understand what is happening, even though it's mostly self explanatory.

**Comment by vzvu3k6k** [comment]
@tjgurwara99
Thanks for review. I've added a comment for `cloneIntSlice` at 651bbbc, but can't think of a good comment for `getSortedVersion`. If you have any good ideas, please let me know.

Now that `go test` runs on CI thanks to #266, I've reduced the slice length of testcases (at 0bbc1db) to avoid timeout on GitHub Actions (happened on https://github.com/TheAlgorithms/Go/runs/1995000745). It would be great if you could review this change as well.

File: sorts/sorts_testcases.go [hunk 1]
```diff
@@ -70,6 +70,15 @@ func makeRandomSignedSlice(size int) []int {
 }
 
 func getSortedVersion(a []int) []int {
```
Comment (tjgurwara99) [inline sorts/sorts_testcases.go:72]:
```suggestion
// getSortedVersion Takes a slice of ints and returns it slice as a sorted (in 
// ascending order) copy of the slice passed as an argument.
func getSortedVersion(a []int) []int {
```

or something along those lines. This function is essentially taking in a slice and sorting it with the in-built sorting algorithm. The in-built algorithms sorts it according to the second argument which is passed as a comparison `lambda` (not sure if its called lambda in Golang - mathematically its a lambda). So we can work with the sorted version of the unsorted random slices that we generated for testing to compare the answer of the algorithms with a properly sorted slice.

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
Good work catching that unintentional sorting!

File: sorts/sorts_testcases.go (hunk 1 above)
Comment (vzvu3k6k) [reply sorts/sorts_testcases.go:74]: Thanks for suggestion! Just committed.

**Comment by tjgurwara99** [comment]
This issue has been resolved recently when I was restructuring the repo. You can close this PR now. Thank you.

**Pull request closed without merging**